Source/WorkersES6/createVectorTileClampedPolylines.js

```diff
@@ -0,0 +1,507 @@
+import AttributeCompression from "../Core/AttributeCompression.js";
+import Cartesian3 from "../Core/Cartesian3.js";
+import Cartographic from "../Core/Cartographic.js";
+import Ellipsoid from "../Core/Ellipsoid.js";
+import IndexDatatype from "../Core/IndexDatatype.js";
+import CesiumMath from "../Core/Math.js";
+import Rectangle from "../Core/Rectangle.js";
+import createTaskProcessorWorker from "./createTaskProcessorWorker.js";
+
+var MAX_SHORT = 32767;
+var MITER_BREAK = Math.cos(CesiumMath.toRadians(150.0));
+
+var scratchBVCartographic = new Cartographic();
+var scratchEncodedPosition = new Cartesian3();
+
+function decodePositionsToRtc(
+  uBuffer,
+  vBuffer,
+  heightBuffer,
+  rectangle,
+  minimumHeight,
+  maximumHeight,
+  ellipsoid,
+  center
+) {
+  var positionsLength = uBuffer.length;
+  var decodedPositions = new Float32Array(positionsLength * 3);
+  for (var i = 0; i < positionsLength; ++i) {
+    var u = uBuffer[i];
+    var v = vBuffer[i];
+    var h = heightBuffer[i];
+
+    var lon = CesiumMath.lerp(rectangle.west, rectangle.east, u / MAX_SHORT);
+    var lat = CesiumMath.lerp(rectangle.south, rectangle.north, v / MAX_SHORT);
+    var alt = CesiumMath.lerp(minimumHeight, maximumHeight, h / MAX_SHORT);
+
+    var cartographic = Cartographic.fromRadians(
+      lon,
+      lat,
+      alt,
+      scratchBVCartographic
+    );
+    var decodedPosition = ellipsoid.cartographicToCartesian(
+      cartographic,
+      scratchEncodedPosition
+    );
+    var rtc = Cartesian3.subtract(
+      decodedPosition,
+      center,
+      scratchEncodedPosition
+    );
+    Cartesian3.pack(rtc, decodedPositions, i * 3);
+  }
+  return decodedPositions;
+}
+
+var previousCompressedCartographicScratch = new Cartographic();
+var currentCompressedCartographicScratch = new Cartographic();
+function removeDuplicates(uBuffer, vBuffer, heightBuffer, counts) {
```

Comment:
This might be handy to have elsewhere in vector tiles too, wherever positions are getting unpacked from those quantized u/v buffers. It'll need a more specific name if it becomes its own file though.